src/sys.h

```diff
@@ -37,7 +37,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 #ifndef SERVER
 int resources_list(struct MEM_PROFILE *, struct VOLUME_PROFILE *);
 #endif
-int get_memory_usage(struct MEM_PROFILE *, char *return_message);
-struct VOLUME_PROFILE *search_volumes_targets(char *return_message);
+int get_memory_usage(struct MEM_PROFILE *, char *);
```

Comment:
I believe that leaving the variables' names is useful, at least with my IDE (JetBrains). But it is just cosmetic...

Reply:
I will change it to add the var names.